server/game/cards/01_SOR/units/AdmiralAckbarBrilliantStrategist.ts

```diff
@@ -0,0 +1,37 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { AbilityContext } from '../../../core/ability/AbilityContext';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { WildcardCardType } from '../../../core/Constants';
+
+export default class AdmiralAckbarBrilliantStrategist extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '0827076106',
+            internalName: 'admiral-ackbar#brilliant-strategist'
+        };
+    }
+
+    /**
+     * Determines the number of units in the ability's target unit arena.
+     * @param context The context from the ability.
+     * @returns The number of units in target units arena.
+     */
+    private getDamageFromContext(context: AbilityContext): number {
+        const arenaName = context.target._location;
+        const arena = context.source.controller.getUnitsInPlay(arenaName);
+        return arena.length;
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'You may deal damage to a unit equal to the number of units your control in its arena.',
```

Comment:
nit: for ability titles our convention right now is to just provide the effect description and not include any cost or "you may" wording since that info will be provided by the context of the UX (e.g., offering a "pass" button or something). So in this case it would just be "Deal damage to a unit equal to the number of units your control in its arena"

also nit, typo: "units your control"